chunker.go

```diff
@@ -131,11 +132,9 @@ func (c *BaseChunker) fillTables() {
 	//
 	// Afterwards a new byte can be shifted in.
 	for b := 0; b < 256; b++ {
-		var h Pol
-
-		h = appendByte(h, byte(b), c.pol)
-		for i := 0; i < windowSize-1; i++ {
-			h = appendByte(h, 0, c.pol)
+		h := Pol(b)
```

Comment:
I'm not sure whether it's worth the trouble to optimize `fillTables()` and its helper functions, as the code only runs a single time.

Reply:
Indeed.